js/remote/index.js

```diff
@@ -24,21 +30,33 @@ process.on('message', process.exit);
 /*
  * if -d or --debug is specified as a command line argument
  * run server on 127.0.0.1 (localhost)
- * return all sensor calls with random data
+ * return all sensor calls with dummy data
  */
-const argv = clp(process.argv);
-const debug = argv['d'] || argv['debug'];
-if (debug) console.log('running in debug mode');
+const args = yargs
+    .usage('Usage: $0 [options]')
+    .version(false)
+    .option('d', {
+        alias: 'debug',
+        desc: 'use fake sensor values instead of real onez',
+        type: 'boolean'
+    })
+    .option('l', {
+        alias: 'local',
+        desc: 'run the server on localhost',
+        type: 'boolean',
+        implies: 'debug'
+    })
+    .alias('h', 'help')
+    .argv;
+
+if (args.debug) console.log('running in debug mode');
 
 // global constants
-// TODO: figure out a more efficient way to get the pi's address
-const piHost = '0.0.0.0';   // this will eventually be the address of the pi we want to host this on
+// TODO: figure out a way to get the pi's address
+const address = args.local ? '127.0.0.1' : '0.0.0.0';
```

Comment:
So `address` is dependent on the value of `args.local`, which sets the output to one of two IP's. What's the benefit of having more than one potential IP?